joeynmt/training.py

```diff
@@ -92,13 +94,12 @@ def __init__(self, model: Model, config: dict) -> None:
         # validation & early stopping
         self.validation_freq = train_config.get("validation_freq", 1000)
         self.log_valid_sents = train_config.get("print_valid_sents", [0, 1, 2])
-        self.ckpt_queue = queue.Queue(
-            maxsize=train_config.get("keep_last_ckpts", 5))
+        self.ckpt_queue = collections.deque(
+            maxlen=train_config.get("keep_last_ckpts", 5))
```

Comment:
Maybe we should change that variable name to "keep_num_checkpoints" to avoid confusion with "save_latest_checkpoint" - what do you think?

Reply:
I think a re-naming of variables would be a good idea, but is there a different name for save_latest_checkpoint that would be good so config files wouldn't have to be updated? I think that would be best for backwards compatibility.

Reply:
Good point. Let's leave it like you proposed then, maybe "save_latest_ckpt" to be consistent with abbreviations? ;) And make sure to write a line in configs/small.yaml to describe the difference, maybe add a point to the FAQ in the docs as well.

Reply:
Sounds good, I'll change the string and add it to the config with an explanation. How do I update the docs? Do they automatically get built by readthedocs.io?

Reply:
Sounds good! Yes they get built automatically once you change something in the ´docs´ directory.